news/news.md

```diff
@@ -1,4 +1,7 @@
 # News
+## 03 September 2017
+* [TenX has become a member of the Enterprise Ethereum Alliance](tenx_03-09-17.md)
```

Comment:
Please, add this news as link and description ( add "## News" ) to corresponding project